docs/java/features/extensions/extension-library/curconv/currency-conversion.mdx

```diff
@@ -13,77 +13,265 @@ keywords:
+:::
+
+You can use the library for the following functions:
+
+- Direct one-to-one conversions, from a source currency to a target currency.
+- Indirect conversions by using inverse rates or a "reference currency".
+- Fixed rate conversions, for when you have the exact currency exchange rate that you want to use in your conversion operations.
+- Non-fixed rate conversions, for when you have a set of exchange rates, as opposed to the exact rate. The library will pick the "best rate" from the set depending on [various factors](#non-fixed-rate).
+
+## Getting Started
 
-## Getting started
+Adding the Currency Conversion library to your list of dependencies will allow you to perform the following tasks:
 
-Edit you `pom.xml` file and add Currency Conversion library to your dependencies.
+- Convert a single or list of amounts from one currency to another by using a fixed rate.
+- Convert a single or list of amounts from one currency to another by using exchange rates provided at runtime.
+- Convert a single or list of amounts from one currency to another by using tenant-specific rates provided at runtime.
+- Configure the conversion process for users or tenants accessing your application.
+
+Add the dependency in your pom.xml file under the existing *dependencies* tag as follows:
 
 ```xml
 <dependency>
    <groupId>com.sap.cloud.sdk.services</groupId>
    <artifactId>currency-conversion-core</artifactId>
-   <!-- Please, use the latest version of the dependency -->
+   <!-- Please use the latest version of the dependency -->
    <version>3.X.X</version>
 </dependency>
 ```
-If your IDE haven't resolved the dependencies run `mvn clean install` in the root directory of you project.
+The library works within the tenant context, therefore, depending on the landscape you operate in, add one of the following dependencies in your pom.xml file, in addition to that of the library's. Adding this additional dependency helps the library detect your tenants automatically. Without it, you would have to manually pass this information. Therefore, we recommend adding one or more of the following dependencies, based on your landscape.
 
-## Features
+If you want to run the library in the Cloud Foundry environment, use the following:
 
-### Convert Currencies
+```xml
+<dependency>
+  <groupId>com.sap.cloud.sdk.cloudplatform</groupId>
+  <artifactId>tenant-scp-cf</artifactId>
+  <!-- Please use the latest version of the dependency -->
+  <version>3.X.X</version>
+</dependency>
+```
 
-#### Get currency exchange rates converted from a base currency to a target currency
+If you want to run the library in the Neo environment, use the following:
 
-```java
-//example code
+```xml
+<dependency>
+  <groupId>com.sap.cloud.sdk.cloudplatform</groupId>
+  <artifactId>tenant-scp-neo</artifactId>
+  <!-- Please use the latest version of the dependency -->
+  <version>3.X.X</version>
+</dependency>  
```

Comment:
@artemkovalyov These dependencies are needed. Otherwise the tenant resolution of the SDK does not work.

Reply:
We could add a hint that adding this dependency is redundant if it is already there. This is the case if the SDK archetypes are used. Otherwise adding them is the right approach

Reply:
We're hesitant to add a line that says, "Don't add these dependencies, if you already have them via Native SDK" because it seems too obvious. Let us know what you think.